dgv/monitoring/DAG-administrateur.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+
+from airflow.models import DAG
+from airflow.operators.python import PythonOperator, ShortCircuitOperator
+from airflow.utils.dates import days_ago
+from datagouvfr_data_pipelines.config import (
+    DATAGOUV_URL,
+    MATTERMOST_DATAGOUV_DATAENG_TEST  # TODO: replace with MATTERMOST_DATAGOUV_ACTIVITES
+)
+from datagouvfr_data_pipelines.utils.mattermost import send_message
+from datagouvfr_data_pipelines.utils.datagouv import get_all_from_api_query
+
+DAG_NAME = "dgv_administrateur"
+
+
+def list_current_admins(ti):
+    # We want the list of all current users with an admin role
+    print(f"Fetching admins from {DATAGOUV_URL}/api/1/users/")
+    users = get_all_from_api_query(f"{DATAGOUV_URL}/api/1/users/?page_size=100")
+    admins = [user for user in users if "admin" in user["roles"]]
+    ti.xcom_push(key="admins", value=admins)
+    return True
+
+
+def publish_mattermost(ti):
+    print("Publishing on mattermost")
+    admins = ti.xcom_pull(key="admins", task_ids="list_current_admins")
+    message = (
+        f":superhero: Voici la liste des {len(admins)} administrateurs actuels de data.gouv.fr"
+    )
+    for admin in admins:
+        message += f"\n* [{admin['first_name']} {admin['last_name']}]({admin['page']})"
+    print(message)
+    send_message(message, MATTERMOST_DATAGOUV_DATAENG_TEST)
+
+
+default_args = {"email": ["geoffrey.aldebert@data.gouv.fr"], "email_on_failure": True}
+
+with DAG(
+    dag_id=DAG_NAME,
+    schedule_interval="0 0 1 1/3 *",
+    start_date=days_ago(0, hour=1),
+    dagrun_timeout=timedelta(minutes=60),
+    tags=["curation", "datagouv"],
+    default_args=default_args,
+    catchup=False,
+) as dag:
+    list_current_admins = ShortCircuitOperator(
```

Comment:
Do you need the `ShortCircuit` here? You can switch to a `PythonOperator` otherwise (and remove `return True` from the callable)